### PR TITLE
kdump: fix guest can not do kdump over ssh

### PR DESCRIPTION
**Pull request by liaolucy**
Change location of SSH connection setup;
 Add check core dump for each panic;

bug id: 1543770

Signed-off-by: Chunyu Liao cliao@redhat.com

### CODE REVIEW

**Comment by hereischen** [comment]
Please provide the test result.

**Comment by liaolucy** [comment]
'1-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.kdump.one_vm.with_ssh' Finshed 2018-03-23 01:19:38 2018-03-23 01:23:11 PASS 212.150027
'2-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.kdump.one_vm.with_ssh.nmi' Finshed 2018-03-23 01:23:11 2018-03-23 01:25:40 PASS 149.625385

File: generic/tests/kdump.py [hunk 1]
```diff
@@ -69,7 +69,6 @@ def kdump_enable(vm, vm_name, crash_kernel_prob_cmd,
     kdump_config = vm.params.get("kdump_config")
     vmcore_path = vm.params.get("vmcore_path", "/var/crash")
     kdump_method = vm.params.get("kdump_method", "basic")
-    kdump_propagate_cmd = vm.params.get("kdump_propagate_cmd")
```
Comment (hereischen) [inline generic/tests/kdump.py]: why did you remove  kdump_propagate_cmd?

Comment (liaolucy) [reply generic/tests/kdump.py]: kdump_propagate_cmd is kdumpctl propagate.
propagate:Helps to setup key authentication for ssh storage since it's impossible to use password authentication during
              kdump.
This key can be use only one time, after trigger kdump and system reboot the key can not be use in rhel7.5.
kdump.one_vm.with_ssh will trigger kdump more than once.

**Comment by liaolucy** [comment]
2-smp_8.4096m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.nmi_bsod_catch' Finshed 2018-03-29 09:46:38 2018-03-29 09:53:19 PASS 400.542337

**Comment by ngu-niny** [comment]
Chunyu, I write down our discussion as follows:
1. Reuse kdump_propagate_cmd if necessary.
2. Make following types of logs clear:
    Context: Setup ssh login without password... --> Crash kernel is not loaded. Trying to load it
3. Check core dump for each vcpu.

**Comment by ngu-niny** [comment]
@liaolucy Test would fail for failing to find the vmcore file, not sure if it's a product bug.
http://10.66.4.244/kvm_autotest_job_log/?jobid=2413920

**Comment by ngu-niny** [comment]
@liaolucy Shall we disable pvpanic in the script?

**Comment by hereischen** [comment]
Please resolve the conflict

**Comment by sathnaga** [comment]
resolve conflict..
